packages/swing-store/src/snapStore.js

```diff
@@ -259,19 +297,51 @@ export function makeSnapStore(
     );
   }
 
-  const sqlDeleteSnapshot = db.prepare(`
+  const sqlGetSnapshotInfo = db.prepare(`
+    SELECT endPos, hash
+    FROM snapshots
+    WHERE vatID = ?
+    ORDER BY endPos DESC
+    LIMIT 1
+  `);
+
+  function getSnapshotInfo(vatID) {
+    return /** @type {SnapshotInfo} */ (sqlGetSnapshotInfo.get(vatID));
```

Comment:
Does this type need to accomodate a return value of `undefined` sometimes? Or, is the return value of `getSnapshotInfo` really a `SnapshotInfo | undefined`? I guess that would do it too.

Reply:
I don't really care, since the cast is just to make TypeScript shut up.